contracts/src/Rollup.sol

```diff
@@ -57,9 +57,12 @@ abstract contract RollupBase is
     IDAProvider public daProvider;
     IVerifier public verifier;
 
+    // Current txBatch serialization version
+    uint8 public constant currentAssertionVersion = 0;
+
     struct AssertionState {
         mapping(address => bool) stakers; // all stakers that have ever staked on this assertion.
-        mapping(bytes32 => bool) childStateHashes; // child assertion vm hashes
+        mapping(bytes => bool) childAssertionData; // child assertion data
```

Comment:
not sure if the key to this mapping should be the versioned blob, or leave it as a state hash

Reply:
I think including the version is probably better, but the map may cost us more because of being arbitrary length byte array? @LEAFERx

Reply:
this is reverted now, based on changes suggested by @ujvl